test/gurka/test_trace_attributes.cc

```diff
@@ -150,4 +150,68 @@ TEST(Standalone, RetrieveTrafficSignal) {
 
   EXPECT_TRUE(edges[1]["end_node"].HasMember("traffic_signal"));
   EXPECT_FALSE(edges[1]["end_node"]["traffic_signal"].GetBool());
-}
\ No newline at end of file
+}
+
+TEST(Standalone, AdditionalSpeedAttributes) {
+  const std::string ascii_map = R"(
+    A---B---C---D---E---F---G
+  )";
+
+  // All edges are set up identically with the same speed attributes
+  gurka::ways ways = {
+      {"AB", {{"highway", "primary"}, {"maxspeed", "60"}}},
+      {"BC", {{"highway", "primary"}, {"maxspeed", "60"}}},
+      {"CD", {{"highway", "primary"}, {"maxspeed", "60"}}},
+      {"DE", {{"highway", "primary"}, {"maxspeed", "60"}}},
+      {"EF", {{"highway", "primary"}, {"maxspeed", "60"}}},
+      {"FG", {{"highway", "primary"}, {"maxspeed", "60"}}},
+  };
+
+  const double gridsize = 100;
```

Comment:
can we make this a LOT coarser? currently there's no test actually triggering any fading.. let's do an approx 30 mins route, so we can see the effect in the test.